draft-ietf-taps-interface.md

```diff
@@ -1109,14 +1109,26 @@ After Stop() is called, the Listener can be disposed of.
 Listener -> ConnectionReceived<Connection>
 ~~~
 
-The ConnectionReceived Event occurs when a Remote Endpoint has established a
+The ConnectionReceived event occurs when a Remote Endpoint has established a
```

Comment:
The terminology section writes events as Events and in other places the word is capitalized too. I suggest to keep this as "Event" now to be reasonably consistent - if you want Events (and, I guess, then also Actions) to be lowercase, this can be discussed, but then let's do this for the whole document at some point.
(Personally, I couldn't care less, but I do want things to be as consistent as possible.)